api/src/main/java/org/openmrs/customdatatype/datatype/ProgramDatatype.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.customdatatype.datatype;
+
+import org.openmrs.Program;
+import org.openmrs.customdatatype.SerializingCustomDatatype;
+
+import com.thoughtworks.xstream.XStream;
+import com.thoughtworks.xstream.io.xml.DomDriver;
+
+public class ProgramDatatype extends SerializingCustomDatatype<Program> {
```

Comment:
Note that other datatypes are annotated with @Component so that they can be autowired by DatatypeService. Please put @Component here and say in javadoc @since 1.12